tests/models/test_image.py

```diff
@@ -343,7 +350,7 @@ def test_image_badness(demo_image):
         assert demo_image.badness == 'Banding, Shaking, Bright Sky'
 
 
-# @pytest.mark.skip( reason="slow" )
+@pytest.mark.skip( reason="slow" )
```

Comment:
Did you leave this uncommented on purpose? For GA it is not that slow...

Reply:
What is GA?

I didn't mean to leave this uncommented; I did it while I was rerunning tests because it was a slow test I didn't really need to wait for.

Reply:
GA = github actions. I meant to ask if the tests were too slow for running on github actions (i.e., too slow in general) or if you were just speeding things up for yourself locally.